ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/storage/keyvalue/RocksDbStorageProvider.java

```diff
@@ -31,38 +32,58 @@
 import org.apache.logging.log4j.Logger;
 
 public class RocksDbStorageProvider {
+  public static long DEFAULT_WORLD_STATE_PREIMAGE_CACHE_SIZE = 5_000L;
```

Comment:
Do we want to expose a `--X` option for this value?  Either as the default or the value that is wired into the world state storage?

Reply:
Maybe?  I think we should be adding those options on an as-needed basis for the time being, so I'll leave it for a potential follow-up PR.